From f904d43382cb001325571aea945f0d071ff214a4 Mon Sep 17 00:00:00 2001
From: Kalman Szalai - KAMI <kami911@gmail.com>
Date: Tue, 02 Jun 2009 20:39:05 +0000
Subject: Default is OK in password dialog

i#102230
fixed by timar

* patches/dev300/apply:
* patches/dev300/uui-passworddlg-ok-default.diff:
---
diff --git a/patches/dev300/apply b/patches/dev300/apply
index 17e7d20..c8a280f 100644
--- a/patches/dev300/apply
+++ b/patches/dev300/apply
@@ -522,6 +522,9 @@ padmin-dialog-resize.diff, i#98164, atimar
 sw-dialogs-resize.diff, i#98164, atimar
 svx-dialogs-resize.diff, i#98164, atimar
 
+# Password dialog - default is OK
+uui-passworddlg-ok-default.diff, i#102230, atimar
+
 [ TemporaryHacks ]
 
 [ RadioButtons ]
diff --git a/patches/dev300/uui-passworddlg-ok-default.diff b/patches/dev300/uui-passworddlg-ok-default.diff
new file mode 100644
index 0000000..fe2e1df
--- a/dev/null
+++ b/patches/dev300/uui-passworddlg-ok-default.diff
@@ -0,0 +1,10 @@
+--- uui/source/passworddlg.src.original	2009-06-02 22:24:16.000000000 +0200
++++ uui/source/passworddlg.src	2009-06-02 22:26:25.000000000 +0200
+@@ -66,6 +66,7 @@
+ 	{
+         Pos = MAP_APPFONT( 27, 56 );
+         Size = MAP_APPFONT( 37, 15 );
++        DefButton = TRUE;
+ 	};
+ 
+ 	CancelButton BTN_PASSWORD_CANCEL
--
cgit v0.8.2
